core/src/main/java/amino/run/kernel/common/ServerInfo.java

```diff
@@ -28,6 +29,10 @@ public String getRegion() {
         return labels.get(KernelServerImpl.REGION_KEY);
     }
 
+    public Map<InetSocketAddress, NodeMetric> metrics;
+
+    public int processorCount; // Available processor count
```

Comment:
@VenuReddy2103 It appears that this is never set anywhere?  So it's always zero?